beprof/example.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+from beprof.curve import Curve
+import numpy as np
+
+url = 'http://berkeleyearth.lbl.gov/auto/Global/Complete_TAVG_complete.txt'
```

Comment:
A good example could be noisy parabola:

`0.01*f(x) + 1 - x^2`

where f(x) gives random real numbers from interval [-1,1]